docs/articles_en/openvino-workflow/generative-ai-models-guide.rst

```diff
@@ -105,16 +32,77 @@ To start using OpenVINO as a backend for Hugging Face, change the original Huggi
     +model = OVModelForCausalLM.from_pretrained(model_id, export=True)
 
 
+Instead of using ``AutoModelForCasualLM`` from the Hugging Face transformers library,
+switch to ``OVModelForCasualLM`` from the optimum.intel library. This change enables
+you to use OpenVINO's optimization features. You may also use other AutoModel types,
+such as ``OVModelForSeq2SeqLM``, though this guide will focus on CausalLM.
+
+By setting the parameter ``export=True``, the model is converted to OpenVINO IR format on the fly.
+
 After that, you can call ``save_pretrained()`` method to save model to the folder in the OpenVINO
 Intermediate Representation and use it further.
 
 .. code-block:: python
 
-    model.save_pretrained(model_dir)
+    model.save_pretrained("ov_model")
+
+This will create a new folder called `ov_model` with the LLM in OpenVINO IR format inside.
+You can change the folder and provide another model directory instead of `ov_model`.
+
+Once the model is saved, you can load it with the following command:
+
+.. code-block:: python
+
+    model = OVModelForCausalLM.from_pretrained("ov_model")
+
+Obtaining OpenVINO Model Object
```

Comment:
This section is unimportant. It does not bring any information that can be used at once. Moreover, it can confuse newcomers.

Reply:
removed